php-http/httplug-bundle/1.6/etc/packages/httplug.yaml

```diff
@@ -0,0 +1,16 @@
+httplug:
+    plugins:
+        logger:
+            enabled: true
```

Comment:
Excellent question. I want to log my HTTP request in production. 

Should I put your suggestion in a comment?